### PR TITLE
fix(db): Export sql and aggregation functions

### LINKED ISSUE #979: bug: `drizzle-orm/expressions` does not include `sql`, `count`, `sum`, etc.
### Provide environment information

-

### Describe the bug

There's been a change to the `db` package recently that exports `*` from `drizzle-orm/expressions` instead of `drizzle-orm`. This doesn't include some of the necessary functions from `drizzle-orm/sql`.

### Link to reproduction

-

### To reproduce

-

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ahkhanjani**
Closes #979

### CODE REVIEW

**Comment by ahkhanjani** [comment]
Exported `alias` because it is used for subqueries. I feel like it should be exported.

File: packages/db/src/index.ts [hunk 1]
```diff
@@ -10,6 +10,9 @@ export const schema = { ...auth, ...post };
 export { mySqlTable as tableCreator } from "./schema/_table";
 
 export * from "drizzle-orm/expressions";
+export * from "drizzle-orm/sql/functions";
+export { sql } from "drizzle-orm/sql/sql";
```
Comment (juliusmarminge) [inline packages/db/src/index.ts:14]: What "extras" are we getting if we just export the entire `/sql` dir?

Comment (john093e) [reply packages/db/src/index.ts:14]: Hello everyone :)

the type SQL could also be added 
export type { SQL } from "drizzle-orm/sql/sql"

Comment (juliusmarminge) [reply packages/db/src/index.ts:14]: prob best to just `export * from "drizzle-orm/sql"`

Comment (juliusmarminge) [inline packages/db/src/index.ts:14]:
```suggestion
export * from "drizzle-orm/sql";
```

Comment (ahkhanjani) [reply packages/db/src/index.ts:14]: > What "extras" are we getting if we just export the entire `/sql` dir?

All of these:

Screenshot 2024-04-10 150700
Screenshot 2024-04-10 150713

None of them seem necessary.

**Comment by ahkhanjani** [comment]
> > What "extras" are we getting if we just export the entire `/sql` dir?
> 
> All of these:
> 
> Screenshot 2024-04-10 150700 Screenshot 2024-04-10 150713
> 
> None of them seem necessary.

Should I go along with your suggestion? @juliusmarminge

**Comment by ahkhanjani** [comment]
I will commit your suggestion. Merge/edit as you see fit.

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**